conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
+pdf.ruling.section.askForReview.solsTeam.email=reviews@hmrc.gov.uk
+pdf.ruling.section.askForReview.solsTeam.emailText=Cyfeiriad e-bost y tîm SOLS yw: <a class="govuk-link" href="mailto:{0}">{1}</a> Os byddwch yn penderfynu defnyddio e-bost i gyfathrebu, gweler yr wybodaeth bwysig ar ddiwedd y llythyr hwn. Os ydych am gysylltu â'r tîm yn ysgrifenedig, y cyfeiriad yw –
+pdf.application.footer.solicitors.office.and.legal.services.address=Swyddfa'r Cyfreithiwr a Gwasanaethau Cyfreithiol<br> Solicitor's Office and Legal Services<br> Cyllid a Thollau EF<br> HM Revenue and Customs<br> BX9 1ZT
+
+pdf.ruling.section.appealForIndependentTribunal.title=Os hoffech apelio ar dribiwnlys annibynnol
+pdf.ruling.section.appealForIndependentTribunal.appealImmediately=Os byddai'n well gennych, gallwch apelio'n uniongyrchol ar dribiwnlys annibynnol. Gallwch hefyd symud ymlaen i apêl tribiwnlys unwaith y byddwch wedi cael canlyniad yr adolygiad ac yn anghytuno â'r penderfyniad.
+pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals=Mae'n rhaid i'r cais gyrraedd Gwasanaeth Llysoedd a Thribiwnlysoedd EF cyn pen 30 diwrnod i ddyddiad y llythyr hwn, neu cyn pen 30 diwrnod i'r dyddiad ar y llythyr sy'n nodi canlyniad yr adolygiad.
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements=Dylai'r cais hwn cynnwys copïau o'r canlynol:
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1=y llythyr hwn, neu'r llythyr canlyniad yr adolygiad
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2=unrhyw asesiad
+pdf.ruling.section.appealForIndependentTribunal.relevantInfo=Os nad ydych yn anfon yr wybodaeth berthnasol, gall y Gwasanaeth Llysoedd a Thribiwnlysoedd EF wrthod eich apêl.
+pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo=I gael rhagor o wybodaeth am dribiwnlysoedd ewch i'r dudalen isod neu ffoniwch 0300 303 5176.
+pdf.ruling.section.appealForIndependentTribunal.page=<a href="www.gov.uk/tribiwnlys-treth" target="_blank">www.gov.uk/tribiwnlys-treth</a>
+
+pdf.ruling.section.appealsAndReviews.title=Rhagor o wybodaeth am apeliadau ac adolygiadau
+pdf.ruling.section.appealsAndReviews.moreInfo=I gael rhagor o wybodaeth am eich hawliau o ran apelio ac adolygiadau, ewch i'r dudalen ganlynol:
+pdf.ruling.section.appealsAndReviews.page=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy</a>
+pdf.ruling.section.appealsAndReviews.copy=Gallwch ofyn am gopi o'r ddogfen HMRC1 drwy ffonio 0300 200 1900.
```

Comment:
Apostrophe is not displayed currently. Please use the curved one for consistency in welsh. Update to `o’r` thanks